src/Psphere.jl

```diff
@@ -62,6 +62,18 @@ function psphere_eval(psphere::AbstractVector{<:Real}, super::Supercell, site_li
     return a
 end
 
+function psphere_eval(psphere::AbstractVector{<:Real}, super::Supercell, site::AbstractVector{AbstractVector{<:Real}})
+    m = maximum(psphere)
+    a = findall(x->x<0.15*m, psphere)
+    if !isempty(a)
+        println("The following sites have Pspheres <15% of the maximum (", m, "):")
+        for n in a
+            println("Site ", n, ": ", @sprintf("Psphere: %.3f", psphere[n]), @sprintf(" at site [%.3f, %.3f, %.3f]", site[1], site[2], site[3]))
```

Comment:
Should this be `site[n][1], site[n][2], site[n][3]` instead? Since you want to print the coordinates of the position vector at index `n`, not the first three coordinate vectors.

Reply:
Great catch!